.github/workflows/run_tests.yml

```
@@ -5,6 +5,12 @@ name: Test astrodb_utils
 
 on:
   workflow_dispatch:
+    inputs:
+      astrodb-template-db-branch:
+        description: 'Branch of astrodb-template-db to use'
+        required: false
+        default: 'main'
+        type: string
   push:
```

```suggestion

  push:
```
Extra line space for readability